firecracker-control/local.go

```diff
@@ -47,6 +50,7 @@ import (
 var (
 	_               fccontrolTtrpc.FirecrackerService = (*local)(nil)
 	ttrpcAddressEnv                                   = "TTRPC_ADDRESS"
+	stopVMInterval                                    = 10 * time.Millisecond
```

Comment:
why is this reasonable?

Reply:
No strong reason regarding the number. Do you think it is too frequent?

Reply:
We talked offline. Summary is that since the process will be SIGKILL if it doesn't exit cleanly this is ok since the time we will be checking for is relatively short.